tools/openapi2jsonschema/examples/airbyte.local/openapi.yaml

```diff
@@ -2688,6 +2688,12 @@ components:
           $ref: "#/components/schemas/ConnectionStatus"
         syncCatalog:
           $ref: "#/components/schemas/AirbyteCatalog"
+        schemaChange:
```

Comment:
Curious why this is lower-case `schemaChange` and the other `openapi.yaml` is upper-case `SchemaChange`

Reply:
I don't really know the difference between these 2 different openapi yamls. Possibly an error to edit them both? In the other one, I added a separate SchemaChange object so that's why it's capitalized